src/archive.rs

```diff
@@ -291,24 +182,53 @@ impl<'a> Archive<Read + 'a> {
         }
         Ok(())
     }
+}
+
+impl<'a, R: Read> Entries<'a, R> {
+    /// Indicates whether this iterator will return raw entries or not.
+    ///
+    /// If the raw list of entries are returned, then no preprocessing happens
+    /// on account of this library, for example taking into accout GNU long name
+    /// or long link archive members. Raw iteration is disabled by default.
+    pub fn raw(self, raw: bool) -> Entries<'a, R> {
```

Comment:
I felt that this was somewhat different because with iterators you typically always consume by-value and don't have the `into_` prefix, but this is indeed also blending in the builder pattern a bit. I wanted to avoid a mutable method as it seems to work out best if iterators are stateless beyond the "cursor they're maintaining" (e.g. as little mutability as possible).